tungsten_tempest_plugin/config.py

```diff
@@ -35,6 +35,9 @@
     cfg.StrOpt('catalog_type',
                default='sdn-l-config',
                help="Catalog type of the SDN service"),
+    cfg.StrOpt('contrail_version',
+               default='3.2',
```

Comment:
I wonder about the process for deciding on what the default version should be, but I have no objection to it being 3.2 for now.

Reply:
since it was tested against '5.0.1', let's have '5.0.1'

Reply:
Reddy wants to use 3.2 or 5.0 as a default value. Let's use 5.0